semeio/jobs/scripts/fm_pyscal.py

```diff
@@ -249,8 +253,7 @@ def _get_interpolation_values(
     """
     # Read all key-value pairs from parameters.txt
     if not os.path.exists(parameters_file_name):
-        _logger.error("%s does not exists", parameters_file_name)
-        raise IOError
+        raise FileNotFoundError(f"{parameters_file_name} does not exist")
```

Comment:
I guess it is fine that we throw a traceback if this is missing as that should really not be the case.